src/pages/de/guides/deploy/gitlab.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Veröffentliche deine Astro-Website mit GitLab Pages
+description: Anleitung zur Veröffentlichung deiner Astro-Website mittels GitLab Pages.
+layout: ~/layouts/DeployGuideLayout.astro
+---
+
+[GitLab Pages](https://pages.gitlab.io/) ermöglichen es dir eine Astro-Website für deine [GitLab](https://about.gitlab.com/) Projekte, Gruppen oder Benutzerkonten bereitzustellen.
+
+:::tip[Auf der Suche nach einem Beispiel?]
+Schau dir [das offizielle Astro-Beispiel für GitLab Pages an](https://gitlab.com/pages/astro)!
+:::
+
+## So funktioniert die Veröffentlichung
+
+1. Setze den Parameter `.site` in `astro.config.mjs` auf den benötigten Wert.
+2. Setze den Parameter `dist` in `astro.config.mjs` auf den Wert `public` und den Parameter `public` in `astro.config.mjs` auf einen neu benannten Ordner, der alles enthält, was sich derzeit in `public` befindet. Der Grund dafür ist, dass `public` ein zweiter Quellordner in Astro ist. Damit du Dateien von `public` veröffentlichen kannst, musst du die entsprechenden Dateien aus einem anderen Ordner holen. Ihre "astro.config.mjs" könnte damit so aussehen:
+
+   ```js
+   export default defineConfig({
+     sitemap: true,
+     site: 'https://astro.build/',
+   });
```

Comment:
Oh my. This code block doesn't contain any of the options discussed above. But this issue is also present in the English source page, so I'm not suggesting to change this right now.